docs/bugs.json

```
@@ -1,4 +1,14 @@
 [
+    {
+        "uid": "SOL-2022-5",
+        "name": "DirtyBytesArrayToStorage",
+        "summary": "Copying ``bytes`` arrays from memory or calldata to storage may result in dirty storage values.",
+        "description": "Copying ``bytes`` arrays from memory or calldata to storage is done in chunks of 32 bytes. Thereby, dirty values in calldata or memory can be written to storage, which may then become observable after a ``.push()`` on the bytes array in storage.",
```

I think we could make it a little more detailed. Especially I'm not sure if "dirty values" on its own is self-explanatory to everyone. How about this version?

```suggestion
        "description": "Copying ``bytes`` arrays from memory or calldata to storage is done in chunks of 32 bytes even if the length is not a multiple of 32. Thereby, extra bytes past the end of the array may be copied from calldata or memory to storage. These dirty bytes were assumed not to be observable without the use of inline assembly. However, a ``.push()`` without arguments to a ``bytes`` array assumes that the storage is already zeroed and only increments the array length, which may result in dirty bytes becoming a part of the array.",
```

---

I went for something in between for now.